runtime/bindings/python/tests/io_test.py

```diff
@@ -84,162 +31,69 @@ def testCreateArchiveFile(self):
             self.assertTrue(file_path.exists())
             self.assertGreater(file_path.stat().st_size, 0)
 
-    def testSaveArchiveFile(self):
-        index = rt.ParameterIndex()
+    def testArchiveFileRoundtrip(self):
```

Comment:
This failed on the Windows CI: https://github.com/openxla/iree/actions/runs/8558397394/job/23452872479#step:9:1860

```
 874/1396 Test  #906: iree/runtime/bindings/python/io_test .........................................................................***Failed    0.40 sec
E....
======================================================================
ERROR: testArchiveFileRoundtrip (__main__.ParameterApiTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\shutil.py", line 618, in _rmtree_unsafe
    os.unlink(fullname)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\tmp7ttrb4du\\archive.irpa'
```

Reply:
Fixed by https://github.com/openxla/iree/commit/11d22592941b934d47289fe03441a3ed07f14f08